### PR TITLE
Create version card component for the new details page

### PR DESCRIPTION
**Pull request by nvisal1**
***Purpose***
This PR adds the version card component to the new details page.
I also created a grid layout for the details page 

Note: This PR was branched from `details/add-to-splash`. This branch contains the details page splash implementation that is not in master yet.

***Story***
This PR addresses Clubhouse story: https://app.clubhouse.io/clarkcan/story/1549/details-page-version-card

***Screenshot***
Screen Shot 2019-12-16 at 6 53 03 PM

### CODE REVIEW

File: src/app/cube/details/components/action-pad/action-pad.component.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.wrapper {
+    padding: 0px 100px;
+    background-color: white;
+    width: 100px;
+    height: 400px;
+    border: solid black;
```
Comment (nrwinner) [inline src/app/cube/details/components/action-pad/action-pad.component.scss]: image

Comment (pzalep1) [reply src/app/cube/details/components/action-pad/action-pad.component.scss]: image

File: src/app/cube/details/details.component.ts [hunk 2]
```diff
@@ -1,31 +1,55 @@
...
-      // this.getLearningObject(username, learningObjectName);
+    this.route.params.subscribe(async ({ username, learningObjectName }: { username: string, learningObjectName: string }) => {
+      await this.getLearningObject(username, learningObjectName);
     });
   }
 
-  async getLearningObject(username: string, cuid: string, version: number) {
+  async getLearningObject(username: string, cuid: string, version?: number) {
     this.loading = true;
-    this.learningObject = await this.learningObjectService.getLearningObject(username, cuid, version);
```
Comment (pzalep1) [inline src/app/cube/details/details.component.ts]: Thank you

File: src/app/cube/details/components/version-card/version-card.component.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<div class='version-card'>
+  <div class='version-card__icon-container'>
+    <i class="fal fa-comment-alt-lines fa-2x"></i>
+  </div>
+  <div class='version-card__text-container'>
+    You're seeing version 6!
+  </div>
+  <div class='version-card__button-container'>
+    <button class='button'>VIEW CHANGE LOGS</button>
```
Comment (pzalep1) [inline src/app/cube/details/components/version-card/version-card.component.html]: We need an aria label here.

File: src/app/cube/details/components/version-card/version-card.component.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<div class='version-card'>
+  <div class='version-card__icon-container'>
+    <i class="fal fa-comment-alt-lines fa-2x"></i>
+  </div>
+  <div class='version-card__text-container'>
```
Comment (pzalep1) [inline src/app/cube/details/components/version-card/version-card.component.html]: p tag please

**Pull request merged**